src/modules/components/proposal/proposalVoting/proposalVotingContainer/proposalVotingContainer.tsx

```diff
@@ -30,7 +34,7 @@ export const ProposalVotingContainer: React.FC<IProposalVotingContainerProps> =
                 <p className="text-base font-normal leading-normal text-neutral-500">{description}</p>
             </div>
             {isMultiStage && (
-                <Accordion.Container isMulti={false} defaultValue={activeStage}>
+                <Accordion.Container isMulti={false} value={activeStage} onValueChange={onStageClick}>
                     {processedChildren.map((child, index) =>
                         React.isValidElement(child)
                             ? React.cloneElement(child, { ...child.props, index, isMultiStage })
```

Comment:
Maybe we set a key more explicitly here? Can we expect the stages to always come in the same order and not be subject to a reorder when using the string index + state as the control?

Reply:
The proposal stages shouldn't get reordered but always follow the same order. As stage names can be duplicated, the `stageIndex` is the only unique value between stages at the moment